lib/index.js

```diff
@@ -8,7 +8,7 @@
  * @typedef {BufferEncoding|{encoding?: null|BufferEncoding, mode: Mode?, flag?: string}} WriteOptions
  *
  * @typedef {string|Uint8Array} Path Path of the file.
- * @typedef {Path|Options|VFile} Compatible Things that can be
+ * @typedef {Path|URL|Options|VFile} Compatible Things that can be
```

Comment:
The linter of my editor reports an error here: `Cannot find name 'URL'`. Maybe it's just a misconfiguration from my side? Or do I need to import the types somehow?

Reply:
TS doesn’t know about the `URL` global in the Node environment. Maybe @ChristianMurphy or @remcohaszing have a nicer solution for this, but I’d suggest `import {URL} from 'url'`!

Reply:
[This solution](https://github.com/remcohaszing/estree-util-value-to-estree/blob/8af30b544dff700f5de255e110d937c4f9cdefca/src/index.ts#L4-L7) should work. I’m not 100% sure how to translate this to TS in jsdoc.

Reply:
@remcohaszing why not `URL` as the type and `import {URL} from 'url'` in the code??

(TS knows about variables, so it knows that `URL` refers to `typeof import('url').URL`

Reply:
You're right. I initially thought this should be isomorphic, but this library is NodeJS specific.

Reply:
So should I do this?
```suggestion
 * @typedef {typeof import('url').URL} URL WHATWG URL
 * @typedef {Path|URL|Options|VFile} Compatible Things that can be
```

Reply:
If that works, sure. If it doesn't, add the actual import in JS: node supports overwriting the global